lib/themes/seekAnz/theme.treat.ts

```diff
@@ -10,6 +10,15 @@ const info = '#9556b7';
 export default makeTreatTheme({
   ...tokens,
   name: 'seekAnz',
+  typography: {
+    fontFamily:
+      'Roboto, "Helvetica Neue", HelveticaNeue, Helvetica, Arial, sans-serif',
+    fontWeight: {
+      regular: 400,
+      medium: 500,
+      strong: 600,
```

Comment:
Just fixed this recently in `seek-style-guide`, might as well fix it here while we're at it.

```suggestion
      strong: 700,
```